source/includes/connection-snippets/x509-snippets/c-connection.c

```diff
@@ -0,0 +1,32 @@
+// begin x509 connection
+#include <mongoc/mongoc.h>
+#include <stdio.h>
+
+int
+main (int argc, char *argv[])
+{
+   mongoc_collection_t *collection;
+   mongoc_client_t *client;
+   int64_t count;
+
+   mongoc_init ();
+
+   client = mongoc_client_new ("mongodb+srv://<cluster-url>/test?authSource=$external&tlsCertificateKeyFile=/etc/certs/mongodb/client.pem&retryWrites=true&w=majority&authMechanism=MONGODB-X509");
+
+   collection = mongoc_client_get_collection (client, "testDB", "testCol");
+   bson_t filter = BSON_INITIALIZER;
+   count = mongoc_collection_count_documents (collection, &filter, NULL, NULL, NULL, NULL);
+   if(count == -1){
```

Comment:
You may want to make the parenthesis spacing consistent. clang-format can do this for you if you have it installed. And run
`clang-format -style=file -i c-connection.c`
Alternatively, since there are only a few inconsistencies, you can update them manually here.

Reply:
Fixed